autogen/code_utils.py

```diff
@@ -235,6 +235,25 @@ def get_powershell_command():
 
 powershell_command = get_powershell_command()
 
+def check_and_update_powershell_execution_policy():
+    policy_check_command = ["powershell", "Get-ExecutionPolicy"]
```

Comment:
instead of "powershell" here, use the variable powershell_command defined above which detects whether the command "powershell"  or "pwsh" is to be used for the system